docs/Kids-First/Setting-up-your-KF-Portal-Permissions/KF_2_Registration.md

```diff
@@ -5,44 +5,48 @@ title: Registration
 Register for Accounts
 =====================
 
-Ultimately, our goal is to analyze existing Kids First data in new ways,
+Our goal is to analyze existing Kids First data in new ways,
 or in new combinations, in order to improve medical outcomes. However,
 before we can start using the data, we need to do a lot of set up.
 
 Why so much setup?
 ------------------
 
-As you likely know, there are many rules and regulations about who can
-use human derived medical data, and for what purposes. Human genomic
-data is no exception. Even if the human data you intend to use is
-'open', that generally doesn't mean you can do anything you want with
-it, just that there are fewer barriers to seeing it. Data curators like
+There are many rules and regulations about who can
+use human derived medical data, and for what purposes. Even 
+'open' human data has controls and simply indicates that that there are fewer barriers to seeing it. Data curators like
 the Kids First DRC are obligated to enforce access controls and data use
 rules to the best of their ability, and that requires end users like us
-to have accounts that are tied to verified identities.
+to have accounts that are tied to verified identities (e.g. ORCID, eraCommons).
 
-Genomic data is also extremely large and unwieldy. For example, the Kids
```

Comment:
I don't know that this paragraph makes sense in the way I intended without the reference to the data size.

Reply:
I'm not sure the intension. This paragraph to me outlined that KF has the catalog and Cavatica has the data and analysis. The size of the data didn't seem relevant (and anybody coming here not knowing that genomic data is presumably lost?).

Reply:
so. Most places are *not* set up like KF. So most researchers will not be expecting that the data and the catalog live in entirely separate places. It's actually super weird. I guess, when you say 'the data and the catalog are in two completely different websites', what I expect most users to immediately think is "WHY WOULD YOU DO THAT IT MAKES EVERYTHING SO HARD", so I was trying to preemptively answer. I guess it's fine tho